bindings/python/wscript

```diff
@@ -295,50 +304,107 @@ int main ()
                 "automatic scanning of API definitions will not be possible" %
                 (pygccxml_version_str,
                  '.'.join([str(x) for x in REQUIRED_PYGCCXML_VERSION])))
-        ns3waf._report_optional_feature(conf, "pygccxml", "Python API Scanning Support", False,
-                                     "pygccxml too old")
+        ns3waf._report_optional_feature(conf, "castxml", "Python API Scanning Support", False,
+                                     "pygccxml Python module too old")
```

Comment:
This failure string (pygccxml) no longer matches what is checked (castxml).

Reply:
Sorry for that Sir. I tried to fix this in 3ebf1aee85776fbf2a85bd1fed6a5959c0d4eb97